src/libstd/sync/rwlock.rs

```diff
@@ -24,6 +24,10 @@ use sys_common::rwlock as sys;
 /// of the underlying data (exclusive access) and the read portion of this lock
 /// typically allows for read-only access (shared access).
 ///
+/// The priority policy of the lock is dependant on the underlying operating
+/// system's implementation and therefore may vary with no guarantees of any
+/// particular policy being used.
```

Comment:
The wording here seems slightly awkward (e.g. "may vary with no guarantees"), perhaps something like:

> The priority policy of the lock is dependent on the underlying operating system's implementation, and this type does not guarantee that any particular policy will be used.